### ISSUE TITLE
Support Python 2.6 in the Windows build system.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3545]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3545|
|Type|enhancement|
|Created|2008-11-17 09:23:05Z|

I'd be so happy.

Attachments:

* [3545.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/73f/73f70a21f80eccdff71d8ed4df0d1f2f57cd555d/193d43dbf14ff296ffa8f5fcde750f5d01e37569.diff) (1802 bytes) - added by therve on 2008-11-17 09:26:01Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3545 3545
type__enhancement enhancement
reporter__therve therve
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1226913785000000 1226913785000000
changetime__1226936188000000 1226936188000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve removed owner|
|-|-|

Done in the attached patch, please review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Isn't this a duplicate of [#2939](https://github.com/twisted/twisted/issues/2939)?

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve commented|
|-|-|

Nope, this one is for **Python** 2.6.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

Ah, of course.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @therve|
|-|-|

I assume you tried out the changes and were able to build an installer for Python 2.6 with them.  If that's the case, then please go ahead and apply these changes.

As an aside, it might be nice if we had a buildbot building installers based on this stuff.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_6"></a>|@therve set status to `closed`|
|-|-|

(In [25388]) Add python 2.6 support to win32 build system.

Author: therve
Reviewer: exarkun
Fixes [#3545](https://github.com/twisted/twisted/issues/3545)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
